handlers/feedback.go

```diff
@@ -64,14 +68,15 @@ func getFeedback(w http.ResponseWriter, req *http.Request, validationErrors []co
 }
 
 // AddFeedback handles a users feedback request
-func (f *Feedback) AddFeedback() http.HandlerFunc {
+func (f *Feedback) AddFeedback(cfg *config.Config) http.HandlerFunc {
 	return dphandlers.ControllerHandler(func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-		addFeedback(w, req, f.Render, f.EmailSender, f.Config.FeedbackFrom, f.Config.FeedbackTo, lang, f.Config.SiteDomain, f.CacheService)
+		addFeedback(w, req, f.Render, f.EmailSender, f.Config.FeedbackFrom, f.Config.FeedbackTo, lang, f.Config.SiteDomain, f.CacheService, cfg)
```

Comment:
you can actually pass f.Config.EnableFeedbackAPI here as this is aprt of the `Feedback` struct which has access to Config.

Reply:
Done